src/components/legend/style.js

```diff
@@ -356,12 +356,27 @@ module.exports = function style(s, gd) {
 
         pts.each(function() {
             var p = d3.select(this);
-            var w = boundLineWidth(undefined, trace.line, MAX_MARKER_LINE_WIDTH, CST_MARKER_LINE_WIDTH);
 
-            p.style('stroke-width', w + 'px')
-                .call(Color.fill, trace.fillcolor);
+            if((trace.boxpoints === 'all' || trace.points === 'all') &&
```

Comment:
@nicolaskruchten do you use other `(box)points` values for your `px` strip traces?

If so, we might need to add logic for `marker.outliercolor`, `marker.line.outliercolor` and `marker.line.outlierwidth` in the block below.

Reply:
might be interesting to add a mode where we show all points but style [suspected]outliers differently - but currently these other modes both hide the non-outlier points entirely - which seems like an awfully bad idea to combine with hiding the box/violin!

Reply:
for `px` I was planning on using `'all'`, because of @alexcjohnson's comment :) so this is looking great to me :)